cli/dts/typescript.d.ts

```diff
@@ -1953,7 +1953,7 @@ declare namespace ts {
         readonly parent: JSDoc;
         readonly name: EntityName;
         readonly typeExpression?: JSDocTypeExpression;
-        /** Whether the property name came before the type -- non-standard for JSDoc, but Typescript-like */
+        /** Whether the property name came before the type -- non-standard for JSDoc, but TypeScript-like */
```

Comment:
This is what exists in the upstream TypeScript projects and is copied directly from their distributions.  If you want to fix this, you will need to raise a PR at https://github.com/microsoft/TypeScript.

Reply:
I understand. Thank you.